admin/load.php

```diff
@@ -503,3 +533,37 @@ function perflab_dismiss_wp_pointer_wrapper() {
 	check_ajax_referer( 'dismiss_pointer' );
 }
 add_action( 'wp_ajax_dismiss-wp-pointer', 'perflab_dismiss_wp_pointer_wrapper', 0 );
+
+/**
+ * Callback function hooked to admin_init to handle WPP inline standalone
+ * plugin actions such as activation and deactivation.
+ *
+ * @since n.e.x.t
+ * @return void
+ */
+function perflab_handle_inline_plugin_actions() {
+	if (
+		! empty( filter_input( INPUT_GET, 'wpp' ) ) &&
+		! empty( filter_input( INPUT_GET, 'action' ) ) &&
+		! empty( filter_input( INPUT_GET, 'plugin' ) ) &&
+		'1' === filter_input( INPUT_GET, 'wpp' ) &&
+		(
+			'deactivate' === filter_input( INPUT_GET, 'action' ) ||
+			'activate' === filter_input( INPUT_GET, 'action' )
+		)
+	) {
+		$referer = wp_get_referer();
+
+		switch ( filter_input( INPUT_GET, 'action' ) ) {
+			case 'activate':
+				activate_plugins( filter_input( INPUT_GET, 'plugin' ) );
+				break;
+			case 'deactivate':
+				deactivate_plugins( filter_input( INPUT_GET, 'plugin' ) );
+				break;
+		}
+		wp_safe_redirect( $referer );
+		exit;
```

Comment:
This would facilitate the function to be unit tested, as a test could filter `wp_redirect` to return false:

```suggestion
		if ( wp_safe_redirect( $referer ) ) {
			exit;
		}
```